extensions/adobe/experience/customerJourneyManagement/messageProfile.example.2.json

```
@@ -0,0 +1,12 @@
+{
+  "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/messageProfileID": "4218b775-bef3-46b2-aee2-7caae052cf95",
+  "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/channel": {
+    "@id": "https://ns.adobe.com/xdm/channels/email"
+  },
+  "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/variant": "A",
+  "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/isSendTimeOptimized": true,
+  "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/isTestExecution": false,
+  "https://ns.adobe.com/experience/customerJourneyManagement/smsChannelContext/inboundMessage": "stop",
+  "https://ns.adobe.com/experience/customerJourneyManagement/smsChannelContext/inboundNumber": "1234",
+  "https://ns.adobe.com/experience/customerJourneyManagement/smsChannelContext/channelType": "MMS"
+}
```

```suggestion
}

```
Add a new line for build checks to pass